engine/fprintd.go

```
@@ -0,0 +1,130 @@
+// nil so that the caller can proceed without a fingerprint gate. This mirrors
+// the macOS Touch ID best-effort posture: protect where possible, but do not
+// hard-fail on machines without biometric hardware.
+//
+// If fprintd is available and the user explicitly fails verification (no
+// match, timeout, etc.) it returns an error and the caller should not release
+// the secret.
+func FprintdVerify(ctx context.Context, reason string) error {
+	_ = reason
+
+	// Quick active check; skip the whole D-Bus dance if the daemon isn't running.
+	if err := exec.Command("systemctl", "is-active", "--quiet", "fprintd.service").Run(); err != nil {
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)